rocky/tests/test_organization.py

```diff
@@ -156,6 +156,24 @@ def test_organization_member_give_and_revoke_clearance_no_action_reloads_page(rf
     assertContains(response, "Grant")
 
 
+def test_organization_member_give_and_revoke_clearance_permissions(rf, superuser_member, redteam_member, client_member):
+    """Redteamers and clients cannot give/revoke clearances or block/unblock users."""
+    for user in [redteam_member.user, client_member.user]:
+        for action in ["give_clearance", "withdraw_clearance", "block", "unblock"]:
```

Comment:
Perhaps use pytest's `parametrize` decorator for this? This would also give more meaningful results about which values cause trouble

Reply:
Thanks for the suggestion! Updated to use `@pytest.mark.parametrize`.